doc/architecture.rst

```diff
@@ -30,58 +30,52 @@ A Ceph Storage Cluster consists of multiple types of daemons:
 - :term:`Ceph Manager`
 - :term:`Ceph Metadata Server`
 
-.. ditaa::
-
-            +---------------+ +---------------+ +---------------+ +---------------+ 
-            |      OSDs     | |    Monitors   | |    Managers   | |      MDS      |
-            +---------------+ +---------------+ +---------------+ +---------------+ 
-
-A Ceph Monitor maintains a master copy of the cluster map. A cluster of Ceph
-monitors ensures high availability should a monitor daemon fail. Storage cluster
-clients retrieve a copy of the cluster map from the Ceph Monitor.
+A Ceph Monitor maintains a master copy of the cluster map. Having more than one
+Ceph monitor in the Ceph cluster ensures high availability in the event that
+one of the monitor daemons fails. The Ceph monitor provides copies of the cluster
```

Comment:
Ceph Monitors maintain the master copy of the cluster map, which they provide to Ceph clients.  Provisioning multiple monitors within a cluster ensure availability in the event that one of the monitor daemons or its host fails.

Reply:
Accepted.